src/getHumanAge.test.js

```diff
@@ -7,4 +7,34 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0', () => {
+    expect(getHumanAge(0, 0))
+      .toStrictEqual([0, 0]);
+  });
+
+  test('boundary values check, minimum age returned', () => {
```

Comment:
```suggestion
  test('boundary values check, minimum age returned', () => {
```
you have an incorrect test name. We write test names like 'should return something when something' or 'should do something when something'. Check all your test names

Reply:
@djtyrf312 thanks so much 4 your time, I'm sorry that I slow down sometimes))